tests/utils.py

```diff
@@ -4,17 +4,21 @@
 
 def z_norm(a, axis=0, threshold=1e-7):
     std = np.std(a, axis, keepdims=True)
-    std[std < threshold] = 1.0
+    std[np.less(std, threshold, where=~np.isnan(std))] = 1.0
```

Comment:
This is some serious NumPy-fu! Very impressive. Thank you, I've learned something new!

Reply:
Rather thank the people from stack overflow :D